templates/api/knowledge-panels/health/ingredients/ingredient.tt.json

```diff
@@ -0,0 +1,37 @@
+{
+    "level": "info",
+    "topics": [
+        "health"
+    ],
+    "size": "small",
+    [% IF panel.ingredient_description OR panel.wikipedia_abstract %]
+    "expanded": false,
+    [% ELSE %]
+    "expanded": false,
+    [% END %]
+    "title_element": {
+        // Note: the app displays line feeds as line feeds...
+        "title": `        [% dash = "-" %]        [% dash.repeat(panel.level) %]        [% display_taxonomy_tag_name("ingredients", panel.ingredient_id) %]        [% IF panel.ingredient.percent.defined %]            ([% round(panel.ingredient.percent) %]%)        [% ELSIF panel.ingredient.percent_estimate.defined %]            ([% round(panel.ingredient.percent_estimate) %]% [% lang("estimate") %])        [% END %]        `,
+    },        
+    "elements": [
+        [% IF panel.ingredient_description -%]
+        {
+            "element_type": "text",
+            "text_element": {
+                "html": `[% panel.ingredient_description %]`
```

Comment:
```suggestion
                "html": `[% edq(panel.ingredient_description) %]`
```

Reply:
The backticks already take care of escaping quotes, so edq() is not needed:
see https://github.com/openfoodfacts/openfoodfacts-server/blob/main/lib/ProductOpener/KnowledgePanels.pm#L298

Reply:
ok cool, that's what I though first, but down in the same file you put edq inside backtick !